job_board/tests.py

```diff
@@ -27,27 +26,22 @@ def test_data_creation(self):
 
         assert Location.objects.filter(name='Eilat', latitude=29.559191910075217, longitude=34.95283437084102).exists()
```

Comment:
Instead of comparing to hardcoded string like 'Eilat', compare to fixture returned data like so:
```py
Location.objects.filter(name=create_data_models_instances[1].name,
```
apply to all relevant cases

Reply:
This could be marked as resolved